Formula/p/purescript.rb

```diff
@@ -2,8 +2,8 @@ class Purescript < Formula
   desc "Strongly typed programming language that compiles to JavaScript"
   homepage "https://www.purescript.org/"
   # TODO: Try to switch `ghc@9.2` to `ghc` when purescript.cabal allows base>=4.17
```

Comment:
maybe create an issue if it does not exist yet.

Reply:
Upstream refuses to upgrade the GHC version unless there is a feature they want or a meaningful bug fix. There's no reason to open an issue here. See https://github.com/purescript/purescript/pull/4510.

Reply:
I definitely disagree with it, but they are entitled to run their project however they like. Not much we can do.